src/company_cmd.cpp

```diff
@@ -876,8 +876,6 @@ CommandCost CmdCompanyCtrl(DoCommandFlag flags, CompanyCtrlAction cca, CompanyID
 
 			if (!(flags & DC_EXEC)) return CommandCost();
 
-			/* Delete any open window of the company */
-			CloseCompanyWindows(c->index);
```

Comment:
I think it should be moved somewhere between  `ChangeOwnershipOfCompanyItems()` call and `delete c`.
Many windows will be closed automatically when removing assets (like depot windows), but you want to make sure to also close possible left over.

Reply:
`delete c` does also call `CloseCompanyWindows` by itself, so I thought it wouldn't be needed.